images/win/scripts/Tests/Wix.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+Import-Module -Name ImageHelpers -Force
+
+
+
+Describe "Wix" {
+    BeforeAll {         
+      $regKey = "HKLM:\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\*"
+      $installedApplications = Get-ItemProperty -Path $regKey
+      $Global:Version = ($installedApplications | Where-Object { $_.DisplayName -and $_.DisplayName.toLower().Contains("wix") } | Select-Object -First 1).DisplayVersion
```

Comment:
Why we need `Global` here?

Reply:
In order to make this variable visible inside test closures. In the other case the variable is invisible inside them

Reply:
Just checked and variable is visible for me.